tracer/src/Datadog.Trace/RemoteConfigurationManagement/ApplyStates.cs

```diff
@@ -0,0 +1,13 @@
+// <copyright file="ApplyStates.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.RemoteConfigurationManagement;
+
+internal static class ApplyStates
+{
+    public const uint UNACKNOWLEDGED = 1;
+    public const uint ACKNOWLEDGED = 2;
+    public const uint ERROR = 3;
+}
```

Comment:
Any reason this isn't an enum?

Reply:
This ends up in json tag sent to the back end, which expects an int. I was sure whether the json would serialize an enum using the labels or values, this seemed the easiest way to ensure it used an int. I'm open to changing to an enum, but there doesn't seem to be a huge amount of difference.

Reply:
Yeah, that's fine. The benefit you get with enums is exhaustiveness checking, and you can configure newtonsoft json to serialize either to int or string, but I'm not worried. Was more interested in why 🙂